src/include/sof/compiler_attributes.h

```diff
@@ -5,7 +5,15 @@
  * Author: Karol Trzcinski <karolx.trzcinski@linux.intel.com>
  */
 
-#ifndef __ZEPHYR__
+#ifdef __ZEPHYR__
+
+/* Get __cache and __force definitions */
+#include <debug/sparse.h>
```

Comment:
Please mention `__CHECKER__` here, see earlier discussions.

Reply:
@marc-hb sorry, could you clarify? Do you mean just extending the comment like "Get __cache and __force definitions if __CHECKER__ is defined" or what do you mean?

Reply:
Yes something like this! It would be enough to make the connection between `git grep __CHECKER__` and sparse. These two dots are very hard to connect otherwise.

Reply:
Btw, looking for debug/sparse.h ? - is this now upstream in Zephyr ?

Reply:
no: https://github.com/zephyrproject-rtos/zephyr/pull/43776